app/code/Magento/Bundle/Test/Mftf/Section/AdminProductFormBundleSection.xml

```diff
@@ -69,6 +69,7 @@
         <element name="priceField" type="input" selector="//div[@data-index='price']//input"/>
         <element name="listedBundleItem" type="text" selector="//tr[@data-repeat-index='0']//div"/>
         <element name="listedBundleItem2" type="text" selector="//tr[@data-repeat-index='2']//div"/>
+        <element name="fieldError" type="text" selector="//div[@data-index='{{fieldIndex}}']//following-sibling::label[@class='admin__field-error']" parameterized="true"/>
```

Comment:
Is this selector also applicable for all the product types? If yes, let's move it to `AdminProductFormSection`.